docker/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG BASE=21.07
+FROM nvcr.io/nvidia/pytorch:${BASE}-py3 as base
+
+FROM base as trtorch-builder-base
+
+RUN apt-get update && apt-get install -y curl gnupg
+RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > /etc/apt/trusted.gpg.d/bazel.gpg
+RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+
+RUN apt-get update && apt-get install -y bazel-4.0.0
+RUN ln -s /usr/bin/bazel-4.0.0 /usr/bin/bazel
+
+# Workaround for bazel expecting both static and shared versions, we only use shared libraries inside container
+RUN cp /usr/lib/x86_64-linux-gnu/libnvinfer.so /usr/lib/x86_64-linux-gnu/libnvinfer_static.a
```

Comment:
For DLFW we should revert this once building is complete

Reply:
This is for trtorch-builder auxiliary container only - neither bazel nor this will go to the Pytorch container